package.py

```diff
@@ -259,6 +333,7 @@ class WindowsPackaging:
     def __init__(self, storage: Storage, env: Environment) -> None:
         self.__storage = storage
         self.__env = env
+        self.__p12 = str(Path(env.temp_dir()).joinpath('certificate.p12').absolute())
```

Comment:
Use the python pathlib module for paths and for the temporary file if what you want is a named temporary file you can use this method: https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile